src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Overview/ThorchainSaversOverview.tsx

```diff
@@ -157,11 +141,55 @@ export const ThorchainSaversOverview: React.FC<OverviewProps> = ({
     selectUnderlyingStakingAssetsWithBalancesAndIcons(state, opportunityDataFilter),
   )
 
+  const txsFilter = useMemo(
+    () => ({
+      accountId,
+      assetId,
+    }),
+    [accountId, assetId],
+  )
+  const pendingTxsLength = useAppSelector(state => selectTxsByFilter(state, txsFilter)).filter(
```

Comment:
your batteries were low when we chatted but lets use`.some()` here

Reply:
Derp, committed it but probably gut rugged by those context switches between branches - 5b13bdbd2744220b46277ac56bf3b48ecc432b53